components/script/dom/resizeobserver.rs

```diff
@@ -223,7 +223,6 @@ enum ObservationState {
 struct ResizeObservation {
     // <https://drafts.csswg.org/resize-observer/#dom-resizeobservation-target>
     // Note: `target` is kept out of here, to avoid having to root the `ResizeObservation`.
-
```

Comment:
Hm, this makes it look like the comment is part of `observed_box`.

Reply:
Rustfmt insists on that. I've moved this comment into the struct comment instead.